### PR TITLE
Three new physiology SOPs

### PR DESCRIPTION
**Pull request by nsamanta**
1. Vanderbilt_physiology_1
2. Vanderbilt_physiology_2
3. Islet physiology protocol 1.html

### CODE REVIEW

**Comment by nsamanta** [comment]
@yimsheng I realized that my Prettier was not working all this while due to some config issue. I fixed it and was able to view all errors. Hoping I fixed `tag` related errors in these 3 files in addition to the HTML codes for symbols that I added. 
Let me know if I still missed something.

**Comment by yimsheng** [comment]
@nsamanta 
image
Is that pH 7.4 here? I think period `.` is safe in html.

File: assets/workflow/Islet Physiology Studies/Islet physiology protocol 1.html [hunk 1]
```diff
@@ -1,94 +1,171 @@
...
+      <br />
+
+      <font size="+1">
+        <li><strong>Set&#45;up</strong></li>
+      </font>
+      <br />
+
+      <ol type="1">
+        <li>
+          Prepare Krebs buffer &#40;pH 7&@46;4&#41; containing 114 mmol&#47;l
```
Comment (yimsheng) [inline assets/workflow/Islet Physiology Studies/Islet physiology protocol 1.html:56]: Is it pH 7.4? I think period '.' is safe in html we can keep it for readability. If you want to convert to encoded character, it should be &#46.
image

File: assets/workflow/Islet Physiology Studies/Islet physiology protocol 1.html [hunk 2]
```diff
@@ -1,94 +1,171 @@
...
+          &#40;pH 7&#46;4&#41;&#44; 0&#46;25&#37; of BSA
+        </li>
+        <li>
+          Divide solution to 5 bottles&#58;
+          <ul style="list-style-type: disc">
+            <li>Buffer A &#40;300 ml&#41;&#58; no substrate</li>
+            <li>
+              Buffer B &#40;200 ml&#41;&#58; 4 mM amino acid mixture<strong
+                ><sup>1</sup> </strong
+              >&#40;AAM&*#41;
```
Comment (yimsheng) [inline assets/workflow/Islet Physiology Studies/Islet physiology protocol 1.html:69]: `)` is &#41 not &*#41

image

**Review by yimsheng — CHANGES_REQUESTED** [review]
those files are still CRLF, please change them to LF

image

**Comment by nsamanta** [comment]
> those files are still CRLF, please change them to LF
> 
> image

@yimsheng Ugh! I'll make these changes.

**Comment by nsamanta** [comment]
@yimsheng Here is the latest commit, with corrections per your comments above. Thanks for catching these errors: https://github.com/faryabiLab/hpap-public/pull/32/commits/2c7be8448a61a3f48892bf1c6fc6dd640c65eb00

**Review by yimsheng — APPROVED** [review]
looks good

**Pull request merged**